### PR TITLE
adding sept heartbeat

### PR DESCRIPTION
**Pull request by jendefig**
September Heartbeat.  Can go out tomorrow or Wednesday. Still needs image.  Expect that later today.

### CODE REVIEW

File: content/blog/2021-09-14-september-21-dvc-heartbeat.md [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+Sahu's 4 part series on the Fundamentals of MLOps
+[Source link](https://ljvmiranda921.github.io/notebook/2021/07/30/data-centric-ml/)_
+
+If you follow the steps through this series, you will learn how to build and
+deploy an end-to-end ML project - all the steps leading to production!
+
+### Miguel Méndez' Tutorial on DVC + MMdetection
+
+This month [Miguel Méndez](https://www.linkedin.com/in/miguel-mendez/) of
+[Gradient](https://www.gradiant.org/en//) brings us a guide on object detection
```
Comment (daavoo) [inline content/blog/2021-09-14-september-21-dvc-heartbeat.md:58]:
```suggestion
[Gradiant](https://www.gradiant.org/en//) brings us a guide on object detection
```

Comment (jendefig) [reply content/blog/2021-09-14-september-21-dvc-heartbeat.md]: Thanks @daavoo!!

**Comment by jendefig** [comment]
This is ready to go.  Don't understand what's happening with the mmdection.png link.  It's visible in the heroku and in my local.  Finding info about error:

https://stackoverflow.com/questions/40182121/whats-the-source-of-error-getaddrinfo-eai-again
https://serverfault.com/questions/1000434/google-compute-vm-getting-error-getaddrinfo-eai-again
https://www.codegrepper.com/code-examples/javascript/remote+ssh+reason%3A+getaddrinfo+EAI_AGAIN+registry.npmjs.org

@shcheklein @rogermparent @julieg18

File: content/blog/2021-09-14-september-21-dvc-heartbeat.md [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+In
+[this new guide,](https://mmeendez8.github.io/2021/08/30/mmdet-dvc-tutorial.html)
+you'll gain a thorough understanding of the steps, have access to
+[his repo](https://github.com/mmeendez8/mmdetection_dvc) for the project, and
+find his thoughts on scaling hyperparameter tuning through this
+[open issue](https://github.com/iterative/dvc/issues/5477#issuecomment-905440724)
+about exeperiments that we are trying to resolve. Join the conversation! We'd
+love your input!
+
+![DVC + MMdetection](uploads/images/2021-09-14/mmdetection.png) _Miguel Méndez'
```
Comment (shcheklein) [inline content/blog/2021-09-14-september-21-dvc-heartbeat.md:74]: I think it should be `/uploads/images...` (mind the leading `/`)

File: content/blog/2021-09-14-september-21-dvc-heartbeat.md [hunk 3]
```diff
@@ -0,0 +1,247 @@
+﻿---
+title: September'21 Heartbeat
```
Comment (shcheklein) [inline content/blog/2021-09-14-september-21-dvc-heartbeat.md:2]: minor: should we start naming heartbeats w/o `dvc` as we do for the gems?

**Review by shcheklein — APPROVED** [review]
One small fix - image URL and it's good to go.

Good stuff, @jendefig !

File: content/blog/2021-09-14-september-21-dvc-heartbeat.md (hunk 3 above)
Comment (jendefig) [reply content/blog/2021-09-14-september-21-dvc-heartbeat.md:2]: Sure!  Will correct!

Comment (jendefig) [reply content/blog/2021-09-14-september-21-dvc-heartbeat.md:2]: @shcheklein  Went to make changes and this happened.  I don't know why these docs are on Sept-Heartbeat

image

Comment (shcheklein) [reply content/blog/2021-09-14-september-21-dvc-heartbeat.md:2]: that's fine ... I suggest we merge it as is (I can fix the link right here from the UI) and we can do the new name later, wdyt?

**Review by shcheklein — APPROVED** [review]

**Pull request merged**